pkg/api/v1/types.go

```diff
@@ -271,6 +276,17 @@ func (v *VirtualMachineInstance) IsCPUDedicated() bool {
 	return v.Spec.Domain.CPU != nil && v.Spec.Domain.CPU.DedicatedCPUPlacement
 }
 
+// WantsToHaveQOSGuaranteed checks if cpu and memoyr limits and requests are identical on the VMI.
+// This is the indicator that people want a VMI with QOS of guaranteed
+func (v *VirtualMachineInstance) WantsToHaveQOSGuaranteed() bool {
+	resources := v.Spec.Domain.Resources
+	if !resources.Requests.Memory().IsZero() && resources.Requests.Memory().Cmp(*resources.Limits.Memory()) == 0 &&
+		!resources.Requests.Cpu().IsZero() && resources.Requests.Cpu().Cmp(*resources.Limits.Cpu()) == 0 {
+		return true
+	}
+	return false
```

Comment:
can it be replaced with
return !resources.Requests.Memory().IsZero() && resources.Requests.Memory().Cmp(*resources.Limits.Memory()) == 0 && !resources.Requests.Cpu().IsZero() && resources.Requests.Cpu().Cmp(*resources.Limits.Cpu()) == 0 
?

Reply:
Done.